test/libdnf5/conf/test_conf.cpp

```diff
@@ -71,3 +71,57 @@ void ConfTest::test_config_pkg_gpgcheck() {
     CPPUNIT_ASSERT_EQUAL(&config_repo.get_pkg_gpgcheck_option(), &config_repo.get_gpgcheck_option());
 #pragma GCC diagnostic pop
 }
+
+libdnf5::ConfigMain copy_construct_config_main(const libdnf5::ConfigMain & config) {
+    return libdnf5::ConfigMain{config};
```

Comment:
OK. RVO (Return Value Optimalization) is mandatory from C++17. The compiler constructs the object directly in the target memory location.